grid.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+
+import Ogre
+
+class GridFloor:
+    def __init__(self, scene, material_name):
+        self.scn_mgr = scene
+        self.material_name = material_name
+        self.drawQueue = 1
+        self.plane_names = ["YZPlane", "XZPlane", "XYPlane"]
+        self.normals = [
+            (1, 0, 0),  # YZPlane normal
+            (0, 1, 0),  # XZPlane normal
+            (0, 0, 1)   # XYPlane normal
+            ]
+        self.axis_colors = [
+            Ogre.ColourValue(1, 0, 0, 1),
+            Ogre.ColourValue(0, 1, 0, 1),
+            Ogre.ColourValue(0, 0, 1, 1)
+            ]
+        self.min_val = -5
+        self.max_val = 5
+
+    def show_plane(self, plane):
+        for p in range(0, 3):
```

Comment:
as the planes dont change often, you could just generate the requested plane here instead of pre-generating them

Reply:
Yes, but the handling of the planes would be more difficult. Having to create and destroy them.

Do they use too much memory?

Reply:
no, that was just an idea..